receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -212,6 +212,10 @@ attributes:
     description: "the name of Kubernetes Node condition. Example: Ready, Memory, PID, DiskPressure"
     type: string
     enabled: true
+  reason:
+    description: "the container status terminated reason"
```

Comment:
If this attribute is only ever going to be used for this specific status, the name should probably be more descriptive. If it may someday be used for more statuses, the description should be more generic.